app/models/order.rb

```diff
@@ -1,8 +1,25 @@
 class Order < ActiveRecord::Base
   belongs_to :user
-  
+
   has_many :item_orders
   has_many :items, through: :item_orders
 
   validates :total_cost, :user_id, presence: true
+
+  def self.complete
+    all.select { |order| order.status == 'complete'}
```

Comment:
Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.